pkgs/by-name/to/toolong/package.nix

```
@@ -0,0 +1,45 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+  testers,
+  toolong,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "toolong";
+  version = "1.4.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "Textualize";
+    repo = "toolong";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-Zd6j1BIrsLJqptg7BXb67qY3DaeHRHieWJoYYCDHaoc=";
+  };
+
+  build-system = [ python3Packages.poetry-core ];
+  dependencies = with python3Packages; [
+    click
+    textual
+    typing-extensions
+  ];
+
+  pythonRelaxDeps = [ "textual" ];
+
+  pythonImportsCheck = [ "toolong" ];
+  doCheck = false; # no tests
+
+  passthru.tests.version = testers.testVersion {
+    package = toolong;
+    command = "${lib.getExe toolong} --version";
+  };
+
+  meta = with lib; {
+    description = "Terminal application to view, tail, merge, and search log files (plus JSONL)";
+    license = licenses.mit;
+    homepage = "https://www.textualize.io";
```

```suggestion
    homepage = "https://github.com/textualize/toolong";
```
https://github.com/Textualize/toolong/blob/5aa22ee878026f46d4d265905c4e1df4d37842ae/pyproject.toml#L9

This one would be more suitable as an actual `toolong` homepage.